### DISCUSSION TITLE
Plain Vim / Vimscript plugin

### DISCUSSION THREAD

**Discussion by chriszarate** [sxyazi/yazi#1040] [2. Show and tell]
There are several `nvim` / Lua plugins linked from the documentation, but none of them work with regular `vim`. I created a very simple one, feedback and issues welcome:

https://github.com/chriszarate/yazi.vim

**Comment by sxyazi** [comment]
Awesome, thanks for creating this plugin! 

Would you like to raise a PR to add it to the [Yazi resource page](https://github.com/yazi-rs/yazi-rs.github.io/blob/main/docs/resources.md) so more people can see it?
